docs/Data_Portal/Users_Guide/mutation_frequency.md

```diff
@@ -127,15 +127,15 @@ The table contains the following fields:
   * __Primary Site__: The anatomical site affected by the disease
   * __# SSM Affected Cases__: The number of affected cases and total number of cases displayed as a fraction and percentage
 
-## Custom Gene and Mutation Filters
+## Custom Mutated Gene and Somatic Mutation Filters
```

Comment:
a bit of a mouthful -- do y'all think it's ok?

Reply:
Looks good to me.  Its descriptive and will show up if I CTRL-F "gene" or "mutation"